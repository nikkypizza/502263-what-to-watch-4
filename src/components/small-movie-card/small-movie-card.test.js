import React from 'react';
import renderer from 'react-test-renderer';
import SmallMovieCard from './small-movie-card.jsx';

const mock = {
  title: `Snatch`,
  imgSrc: `src/image.jpeg`,
  linkURL: `https://index.html`,
  onClick: () => {},
  onMouseEnter: () => {},
  onMouseLeave: () => {}
};

describe(`SmallMovieCard Component`, () => {
  it(`renders`, () => {
    const tree = renderer.create(
        <SmallMovieCard {...mock} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
