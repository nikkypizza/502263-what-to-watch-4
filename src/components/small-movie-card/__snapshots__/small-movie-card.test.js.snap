// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallMovieCard Component renders 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="The Grand Budapest Hotel"
      height="175"
      src="img/the-grand-budapest-hotel-poster.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
      onClick={[Function]}
    >
      The Grand Budapest Hotel
    </a>
  </h3>
</article>
`;
